qemu/tests/timerdevice_boot.py

```diff
@@ -141,19 +147,18 @@ def update_clksrc(session, clksrc):
     if params["os_type"] == "linux":
         error_context.context("Check the hardware time on guest", logging.info)
         guest_hwtime = get_hwtime(session)
+        drift = abs(float(host_time) - float(guest_hwtime))
+        if drift > timerdevice_drift_threshold:
+            test.fail("The guest's hardware time is different with"
+                      " host's system time. Host time: '%s', guest time:"
+                      " '%s'" % (host_time, guest_hwtime))
```

Comment:
I prefer to add more info such as "before reboot"  to distinguish this line and L187.  (also this error message have 'system time' ).

Reply:
@yanan-fu Thanks, we can get reboot info from context info "Sleep '%s' secs before reboot".

Reply:
Yeah, make sense.